modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc

```diff
@@ -598,6 +600,27 @@ be ignored. This flag depends on the `wnd.activity.registerExtension` and has th
 'wnd.activity.sendToSharees' => false,
 ....
 
+=== Make the group membership component to assume the ACL contains a user
+The WND app doesn't have knowledge about the users or groups associated to ACLs. This
+means that an ACL containing "admin" might refer to a user called "admin" or a
+group called "admin". By default, the group membership component consider the ACLs to
+target groups, and as such, it will try to get the information of such group. This
+works fine if the majority of the ACLs target groups. If the majority of the ACLs
+contains users, this might be problematic. The cost of getting the information of a
+group is usually higher than getting the information of a user. As such, this option
+make the group membership component to assume the ACL contains a user, and check whether
+there is a user in ownCloud with such name first. If the name doesn't refer to a user,
+it will get the group information. Note, that this will have performance implications
+if the group membership component can't discard users in a good number of cases. It is
```

Comment:
```suggestion
makes the group membership component assume the ACL contains a user and checks whether
there is a user in ownCloud with such a name first. If the name doesn't refer to a user,
it will get the group information. Note that this will have performance implications
if the group membership component can't discard users in a large number of cases. It is
```

Reply:
Is "can't" correct in the last sentence? I'd assume it should read "can".